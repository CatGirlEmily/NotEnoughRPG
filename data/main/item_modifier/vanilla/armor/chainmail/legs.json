[
  {
    "function": "minecraft:set_components",
    "components": {

      "minecraft:custom_data": "{CHAINMAIL_LEGGINGS:true,rarity:true}"
    },
    "conditions": []
  },
  {
    "function": "minecraft:set_lore",
    "entity": "this",
    "lore": [
      "§7Defense: §a+20",
      " ",
      "§a§lUNCOMMON LEGGINGS"
    ],
    "mode": "replace_all"
  }
]