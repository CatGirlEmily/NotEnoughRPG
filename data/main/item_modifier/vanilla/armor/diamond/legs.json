[
  {
    "function": "minecraft:set_components",
    "components": {

      "minecraft:custom_data": "{DIAMOND_LEGGINGS:true,rarity:true}"
    },
    "conditions": []
  },
  {
    "function": "minecraft:set_lore",
    "entity": "this",
    "lore": [
      "§7Defense: §a+30",
      " ",
      "§a§lUNCOMMON LEGGINGS"
    ],
    "mode": "replace_all"
  }
]