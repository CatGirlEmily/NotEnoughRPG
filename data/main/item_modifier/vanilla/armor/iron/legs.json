[
  {
    "function": "minecraft:set_components",
    "components": {

      "minecraft:custom_data": "{IRON_LEGGINGS:true,rarity:true}"
    },
    "conditions": []
  },
  {
    "function": "minecraft:set_lore",
    "entity": "this",
    "lore": [
      "§7Defense: §a+25",
      " ",
      "§f§lCOMMON LEGGINGS"
    ],
    "mode": "replace_all"
  }
]