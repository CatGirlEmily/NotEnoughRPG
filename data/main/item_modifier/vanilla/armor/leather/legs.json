[
  {
    "function": "minecraft:set_components",
    "components": {

      "minecraft:custom_data": "{LEATHER_LEGGINGS:true,rarity:true}"
    },
    "conditions": []
  },
  {
    "function": "minecraft:set_lore",
    "entity": "this",
    "lore": [
      "§7Defense: §a+10",
      " ",
      "§f§lCOMMON LEGGINGS"
    ],
    "mode": "replace_all"
  }
]