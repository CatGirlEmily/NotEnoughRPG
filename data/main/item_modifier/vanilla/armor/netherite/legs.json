[
  {
    "function": "minecraft:set_components",
    "components": {

      "minecraft:custom_data": "{NETHERITE_LEGGINGS:true,rarity:true}"
    },
    "conditions": []
  },
  {
    "function": "minecraft:set_lore",
    "entity": "this",
    "lore": [
      "§7Defense: §a+40",
      " ",
      "§5§lEPIC LEGGINGS"
    ],
    "mode": "replace_all"
  }
]