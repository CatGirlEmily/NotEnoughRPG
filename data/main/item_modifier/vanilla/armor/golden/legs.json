[
  {
    "function": "minecraft:set_components",
    "components": {

      "minecraft:custom_data": "{GOLDEN_LEGGINGS:true,rarity:true}"
    },
    "conditions": []
  },
  {
    "function": "minecraft:set_lore",
    "entity": "this",
    "lore": [
      "§7Defense: §a+15",
      " ",
      "§f§lCOMMON LEGGINGS"
    ],
    "mode": "replace_all"
  }
]